image_ocr.go

```diff
@@ -25,22 +19,14 @@ func ConvertImage(r io.Reader) (string, map[string]string, error) {
 	defer f.Done()
 
 	meta := make(map[string]string)
-	out := make(chan string, 1)
-
-	// TODO: Why is this done in a separate goroutine when ConvertImage blocks until it returns?
-	go func(file *LocalFile) {
-		langs.RLock()
```

Comment:
It looks to me as if Gosseract would only work one call at a time per language, so we implemented locking which you've removed here (intentionally?). Not sure about why it was done in a goroutine though, would have to look into it.

Reply:
yep cool, i put it back.